webpack/global_test_setup.js

```diff
@@ -6,9 +6,28 @@ import nock from 'nock';
 // output and traceback for actual error.
 const originalConsoleError = global.console.error;
 global.console.error = (error, stack) => {
-  originalConsoleError(error); // ensure error is printed to console
+  originalConsoleError(error); // ensure error is printed to console , comment
+  // in case of ouia-id check
+
+  /* Uncomment block below to filter out PF4 ouiaId errors */
+  /*
+  const ignore = ['in ForwardRef (created by Tabs)',
+     'in Button (created by Label)', 'created by _default',
+     'created by PaginationOptionsMenu', 'created by TypeAheadItems',
+     'in Modal (created by Wizard)',
+     'created by WizardHeader', 'created by Navigation',
+     'The prop `ouiaId` is marked as required in `RowWrapper`',
```

Comment:
RowWrapper is specifically listed as a supported component, so it's on the list in ouia_check.js. If it's not useful to have it for some reason, should probably just delete it there.

Reply:
We don't use RowWrapper anywhere..yet..It's called by TableComposable or some other component we use. I don't know if we'll start using it anytime soon..Can do it either way.

Reply:
I'd comment it out in ouia_check.js then